src/_shared/components/MarkdownPreview/MarkdownPreview.tsx

```diff
@@ -65,7 +65,7 @@ export const MarkdownPreview: React.FC<MarkdownPreviewProps> = (
         </Box>
       </TabPanel>
       <Link
-        href="https://getpocket.atlassian.net/wiki/spaces/PE/pages/2105606280/Pocket+Markdown"
+        href="https://mozilla-hub.atlassian.net/wiki/spaces/PE/pages/390636109/Pocket+Markdown"
```

Comment:
I'd move this page to a different space but don't have access privileges to do so. So for now, it stays in the Pocket space - I just updated the link as the old one was not redirecting (or even working at all).